cmake/templates/TesseractConfig.cmake.in

```diff
@@ -7,39 +7,21 @@
 #    In your CMakeLists.txt, add these lines:
 #
 #    find_package(Tesseract REQUIRED)
-#    include_directories(${Tesseract_INCLUDE_DIRS})
-#    target_link_libraries(MY_TARGET_NAME ${Tesseract_LIBRARIES})
+#    target_link_libraries(MY_TARGET_NAME Tesseract::libtesseract)
 #
 #    This file will define the following variables:
 #      - Tesseract_LIBRARIES             : The list of all imported targets for OpenCV modules.
 #      - Tesseract_INCLUDE_DIRS          : The Tesseract include directories.
-#      - Tesseract_VERSION               : The version of this Tesseract build: "@VERSION_PLAIN@"
-#      - Tesseract_VERSION_MAJOR         : Major version part of Tesseract_VERSION: "@VERSION_MAJOR@"
-#      - Tesseract_VERSION_MINOR         : Minor version part of Tesseract_VERSION: "@VERSION_MINOR@"
 #
 # ===================================================================================
 
 include(${CMAKE_CURRENT_LIST_DIR}/TesseractTargets.cmake)
 
 find_package(Leptonica REQUIRED)
 
-# ======================================================
-#  Version variables:
-# ======================================================
-
-SET(Tesseract_VERSION           @VERSION_PLAIN@)
-SET(Tesseract_VERSION_MAJOR     @VERSION_MAJOR@)
-SET(Tesseract_VERSION_MINOR     @VERSION_MINOR@)
-
-# ======================================================
-# Include directories to add to the user project:
-# ======================================================
-
-# Provide the include directories to the caller
-set(Tesseract_INCLUDE_DIRS @INCLUDE_DIR@)
-
-# ====================================================================
-# Link libraries:
-# ====================================================================
+@PACKAGE_INIT@
 
+set_and_check(Tesseract_INCLUDE_DIR "@PACKAGE_INCLUDE_DIR@")
```

Comment:
This is broken since `INCLUDE_DIR` contains _multiple_ paths and this macro expects the variable to contain _one_ path. This breaks `find_package` outright.